src/defs_urls.h

```diff
@@ -4,6 +4,9 @@
 #define MIXXX_WEBSITE_SHORT_URL "www.mixxx.org"
 #define MIXXX_SUPPORT_URL       "https://www.mixxx.org/support/"
 #define MIXXX_TRANSLATION_URL   "https://www.transifex.com/projects/p/mixxxdj/"
+#define MIXXX_DONATE_URL              \
+    "https://www.paypal.com/cgi-bin/" \
+    "webscr?cmd=_s-xclick&hosted_button_id=QSFMYWN2B3JD2"
```

Comment:
```suggestion
#define MIXXX_DONATE_URL         "https://mixxx.org/donate"
```

after https://github.com/mixxxdj/website/pull/238 has been merged

Reply:
Unless Mixxx is transformed into a legal entity, accepting ``donations`` raises more questions than answers. It is a gray area at least.

Reply:
Mixxx always took donations, I can even see a donation button on our website in 2008 when I use the Wayback Machine. I do agree that we should think about founding an e.V. or joining the SFC, but I don't think this should hold back this PR.

Maybe you want to share your thoughts on these Zulip threads:
- https://mixxx.zulipchat.com/#narrow/stream/109121-core-team/topic/Conservancy
- https://mixxx.zulipchat.com/#narrow/stream/109121-core-team/topic/Mixxx.20e.2EV.2E